finality-aleph/src/session.rs

```diff
@@ -0,0 +1,50 @@
+use super::*;
```

Comment:
Could you make the imports here explicit? Star imports are not particularly readable.

Reply:
Done.